examples/pstore/pscli.go

```diff
@@ -0,0 +1,117 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package main
+
+import (
+	"fmt"
+	"storj.io/storj/pkg/pstore"
+	"github.com/urfave/cli"
+	"github.com/zeebo/errs"
+	"log"
+	"os"
+	"sort"
```

Comment:
dumb nit about import order again. if you separate the order with newlines, gofmt won't reorder the stanzas, so

```
import (
	"fmt"
	"log"
	"os"
	"sort"

	"github.com/urfave/cli"
	"github.com/zeebo/errs"

	"storj.io/storj/pkg/pstore"
)
```